internal/api/start.go

```diff
@@ -41,23 +37,10 @@ func Start(ctx context.Context, listenAddress, dataDIR string, standalone bool,
 		e.Logger.SetLevel(log.INFO)
 	}
 
-	var maxJobsInt int
-	if maxJobs != "" {
-		var err error
-		maxJobsInt, err = strconv.Atoi(maxJobs)
-		if err != nil {
-			e.Logger.Error("Failed to parse MAX_JOBS: ", err)
-			return err
-		}
-	}
-
-	if maxJobsInt == 0 {
-		maxJobsInt = 10
-		e.Logger.Warn("MAX_JOBS is not set, using default of 10")
-	}
+	maxJobs, _ := config.GetInt("max_jobs", 10)
```

Comment:
note for self - review why `GetInt` has two return types but `GetString` has one...

Reply:
either that, or actually use structured config (which will be its own huge refactor)